lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MarkdownUtils.java

```diff
@@ -0,0 +1,19 @@
+package org.eclipse.lemminx.maven;
```

Comment:
Missing copyright header.

Reply:
[Here's one](https://github.com/eclipse/lemminx-maven/blob/master/lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenPluginUtils.java#L1) you could copy and paste, for reference.